content/docs/guides/prisma.md

```diff
@@ -38,26 +38,43 @@ To connect to Neon from Prisma:
-If connection pooling is enabled for your Neon project, which is the default, append `?pgbouncer=true` to the connection string, as described in the [Prisma documentation](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer#add-pgbouncer-to-the-connection-url).
-
-For information about enabling and disabling connection pooling for your Neon project, see [Connection pooling](../../get-started-with-neon/connection-pooling/).
-</Admonition>
+  ```shell
+  DATABASE_URL="postgres://<user>:<password>@<endpoint_hostname>:5432/neondb"
+  ```
 
 where:
 
 - `<endpoint_hostname>` the hostname of the branch endpoint, which is found on the Neon **Dashboard**, under **Connection Settings**.
 - `<user>` is the database user, which is found on the Neon Console **Dashboard** tab, under **Connection Details**.
 - `<password>` is the database user's password, which is provided to you when you create a Neon project.
 
+<Admonition type="note">
+Neon enables connection pooling by default using PgBouncer. Using Prisma Client in a PgBouncer-enabled environment from a serverless function may require adding the `?pgbouncer=true` flag to your connection URL. For example:
+
+```text
+postgres://<user>:<password>@<endpoint_hostname>:5432/neondb?pgbouncer=true
+```
+
+For more information, refer to the [Prisma documentation](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer#add-pgbouncer-to-the-connection-url).
+</Admonition>
+
 ## Configure a shadow database for Prisma Migrate
 
 Prisma Migrate is a migration tool that allows you to easily evolve your database schema from prototyping to production. Prisma Migrate requires a shadow database to detect schema drift. This section describes how to configure a second Neon database, which is required to run the `prisma migrate dev` command.
 
+<Admonition type="note">
+Neon enables connection pooling by default using PgBouncer. Prisma Migrate requires a direct connection to the database and currently does not support connection pooling with PgBouncer. Attempting to run Prisma Migrate commands in an environment that uses PgBouncer for connection pooling results in the following error:
+
+```text
+Error: undefined: Database error
+Error querying the database: db error: ERROR: prepared statement "s0" already exists
+```
+
+If you encounter this error, try disabling connection pooling for your Neon project. See [Configure connection pooling](../../connect/connection-pooling/#configure-connection-pooling).
+
+For more information about this issue, refer to the [Prisma documentation](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management/configure-pg-bouncer#add-pgbouncer-to-the-connection-url).
+</Admonition>
+
 To configure a shadow database:
 
 1. Create a second Neon project and make sure to copy the connection string. Refer to [Create a Neon project](#create-a-neon-project) for instructions.
```

Comment:
Does this mean Prisma users can't use us since we offer only 1 project in the free tier?